laurent22/joplin
# Chore: Update KaTeX asset files

# Summary

https://github.com/laurent22/joplin/commit/2a74f60812aa0da8d6a55afbea224793939b9b8c upgraded KaTeX without also updating its associated plugin asset files. This pull request updates them.

# Testing plan

1. Start Joplin Web (`cd packages/app-mobile && yarn build && yarn serve-web`).
2. Verify that a note with the content `$\KaTeX$` renders correctly.
